### PR TITLE
Completed tasks

### PR DESCRIPTION
**Pull request by mamkhari**
This PR does the following:
- Ticks-off the completed tasks

### CODE REVIEW

File: ToDo.md [hunk 1]
```diff
@@ -150,8 +150,37 @@
...
 
 - Software Engineering Practices Part 1
 - Software Engineering Practices Part 2
 - Introduction to Object-Oriented Programming
+
+![AWS Machine Learning Foundations course](../home/lebukani/Pictures/AWS.png)
+
+### 30 June
+
+- [x] Meet with Bulelani to take me through [MT-1498](https://skaafrica.atlassian.net/secure/RapidBoard.jspa?rapidView=40&projectKey=MT&modal=detail&selectedIssue=MT-1498)
```
Comment (mmphego) [inline ToDo.md:163]: Perhaps don't share private links, the ticket number should suffice

File: ToDo.md [hunk 2]
```diff
@@ -150,8 +150,37 @@
...
+
+- - -
+
+## July 2020
+
+### 01 - 04 July
+
+- [x] Research about graphical user interface
```
Comment (mmphego) [inline ToDo.md:171]: Links of stuff you researched would be nice.

File: ToDo.md [hunk 3]
```diff
@@ -150,8 +150,37 @@
...
+- [x] Go through[Katgui tour](https://github.com/ska-sa/katgui/tree/master/tour)
```
Comment (mmphego) [inline ToDo.md:172]: How did you find the interactive tour? Any improvements needed?

File: ToDo.md [hunk 4]
```diff
@@ -150,8 +150,37 @@
...
+
+### 06 - 07 July
+
+- [x] Read about [KatGUI Architecture](https://docs.google.com/document/d/1l1GKSMwo-rjKBnHmuUDG2F7fsRumGrsmK6PguxjiQig/edit)
```
Comment (mmphego) [inline ToDo.md:176]: Don't share confidential info on git

File: ToDo.md [hunk 5]
```diff
@@ -150,8 +150,37 @@
...
+
+### 08 - 09 July
+
+- [x] Read about AQF tests via [CAM Crib Notes](https://docs.google.com/document/d/10h_caF8k39i-xOieulNahip53esZgRKxpqBjd096Gss/edit#heading=h.9cupa5n6e807)
+
```
Comment (mmphego) [inline ToDo.md:181]: Ditto!

File: ToDo.md [hunk 6]
```diff
@@ -150,8 +150,37 @@
...
+### 10 July
+
+- [x] Read about nose for Python testing
```
Comment (mmphego) [inline ToDo.md:184]: Where did you read this from? Links would be nice - that way when you get stuff you can refer back to your notes.
Try to be as detailed as possible with your note or else 3 months from now you wont have any ideas of what you did let alone how to fix a certain bug etc.

File: ToDo.md (hunk 1 above)
Comment (mamkhari) [reply ToDo.md:163]: Oh, Okay thanks.

File: ToDo.md (hunk 4 above)
Comment (mamkhari) [reply ToDo.md:176]: Got it.

File: ToDo.md (hunk 1 above)
Comment (mamkhari) [reply ToDo.md:163]: Should I put the Jira ticket number, and not the link to the ticket?

Comment (mmphego) [reply ToDo.md:163]: Yes please, only the ticket number

File: ToDo.md (hunk 6 above)
Comment (mamkhari) [reply ToDo.md]: Added the links.

File: ToDo.md (hunk 3 above)
Comment (mamkhari) [reply ToDo.md]: It is straight to the point, short and easy to follower especially for beginners. It looks good to me.

**Pull request closed without merging**